### DISCUSSION TITLE
Mode shift / shift button function?

### DISCUSSION THREAD

**Discussion by zen7type** [Valkirie/HandheldCompanion#975] [Q&A]
Question, does HC offer a mode shift function for the controller button mapping? Where you hold down a button (usually a back button) and it changes the functions / mapping of the other buttons, including the direction pad. So it doubles the functions you can get on your controller. The steam deck and Ally offer this in their standard controller mapping settings but the Legion Go does not -- which is why I'm looking for options to add this functionality to the Go. Thanks.

**Comment by CasperH2O** [comment]
Hey @zen7type , HC does not have such a functionality, nor is it planned for the short term.
